src/vs/workbench/parts/scm/electron-browser/scmViewlet.ts

```diff
@@ -813,7 +816,10 @@ export class RepositoryPanel extends ViewletPanel {
 		anyEvent<any>(onKeyUp, onMouseUp)(() => this.inputBox.validate(), null, this.disposables);
 
 		this.inputBox.value = this.repository.input.value;
-		this.inputBox.onDidChange(value => this.repository.input.value = value, null, this.disposables);
+		this.inputBox.onDidChange(value => {
+			this.storageService.store(RepositoryPanel.COMMIT_MESSAGE_STORAGE_KEY, value, StorageScope.WORKSPACE);
```

Comment:
Shouldn't there be a matching `remove` call for this call to `store`? The value should be removed after the commit happens, right?